### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by ImtiazTonmoy290** [Magisk-Modules-Repo/foxy-boot#4]
Foxy is not installing in Poco x3 android 11 and show this.

![Uploading Screenshot_20210712-195818.png…]()

**Comment by Noorquacker** [comment]
The image doesn't have a link. Can you attach the image on GitHub?

**Comment by ghost** [comment]
![Screenshot_20211217-141808](https://user-images.githubusercontent.com/80021666/146516735-27987a25-a6ef-495d-89ff-1aa55074d47f.jpg)

**Comment by mchiappinam** [comment]
Same here

**Comment by mchiappinam** [comment]
- Copying zip to temp directory
- Installing foxy-boot.zip
- Device is system-as-root
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: service.sh
  inflating: module.prop
  inflating: install.sh
*******************************
           Foxy Boot           
*******************************
******************************
Powered by Magisk (@topjohnwu)
******************************
- Extracting module files
Archive:  /data/user/0/com.topjohnwu.magisk/cache/flash/install.zip
  inflating: bootanimation-arm64-v8a
- Checking the binary
cannot locate "_ZN7android21SurfaceComposerClient17getBuiltInDisplayEi"
! Unable to link the executable!
! Installation failed

**Comment by paulsybrandy** [comment]
Yeah I am getting the exact same results. Not sure how to rectify the "Unable to line executable" error.

**Comment by bphd** [comment]
Same @kitsunyan
